languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -149653,6 +149811,94 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+        <rule id="PRENDRE_ACTION" name="prendre action-agir" tags="picky" default="temp_off">
+            <pattern>
+                <token inflected="yes">prendre</token>
+                <token>action</token>
+            </pattern>
+            <message>Une autre expression semble plus appropriée.</message>
+            <suggestion><match no="1" postag="(V.*)" postag_regexp="yes" postag_replace="$1">agir</match></suggestion>
+            <example correction="agissons">Et surtout <marker>prenons action</marker>.</example>
+	    </rule>
+	    <rulegroup id="ADMIN_PRESIDENT" name="administration-gouvernement president" default="temp_off">
+		    <rule>
+			    <antipattern>
+				    <token>administration</token>
+				    <token regexp="yes">de|du|d'|des</token>
+			    </antipattern>
+			    <antipattern>
+				    <token>administration</token>
+				    <token postag="M nonfin" />
+			    </antipattern>
+	            <pattern>
+	                <marker>
+	                    <token skip="-1">administration</token>
+	                </marker>
+	                <token regexp="yes">Washington|Adams|Jefferson|Madison|Monroe|Jackson|Van Buren|Harrison|Tyler|Polk|Eisenhower|Kennedy|Johnson|Nixon|Ford|Carter|Reagan|Bush|Clinton|Obama|Trump|Biden</token>
+	            </pattern>
+	            <message>Une autre expression semble plus appropriée.</message>
+	            <suggestion>gouvernement de</suggestion>
+	            <example correction="gouvernement de">C'est l'<marker>administration</marker> Obama.</example>
+	            <example correction="gouvernement de">C'est l'<marker>administration</marker> George H. W. Bush.</example>
+		    </rule>
+		    <rule>
+	            <antipattern>
+				    <token>administration</token>
+				    <token regexp="yes">d'|des</token>
+				    <token regexp="yes">services|affaires</token>
+			    </antipattern>
+			    <antipattern>
+				    <token>administration</token>
+				    <token>du</token>
+				    <token>Washington</token>
+				    <token>Post</token>
+			    </antipattern>
+			    <antipattern>
+				    <token regexp="yes">conseils?</token>
+				    <token regexp="yes">d'|de</token>
+				    <token min="0">l'</token>
+				    <token>administration</token>
+				    <example>Le conseil d'administration.</example>
+				    <example>Les conseils d'administration.</example>
+				    <example>À partir de 1989, elle est membre du conseil d'administration d'International Campaign for Tibet à Washington DC.</example>
+		    	</antipattern>
+			    <pattern>
+	                <marker>
+	                    <token>administration</token>
+	                </marker>
+	                <token skip="-1" regexp="yes">de|du|d'|des</token>
+	                <token regexp="yes">Washington|Adams|Jefferson|Madison|Monroe|Jackson|Van Buren|Harrison|Tyler|Polk|Eisenhower|Kennedy|Johnson|Nixon|Ford|Carter|Reagan|Bush|Clinton|Obama|Trump|Biden</token>
```

Comment:
same indentation issue (maybe it's intended, but unusual :))

Reply:
see above. Will remove first part of President van Buren's name.